.github/pull-request-template.md

```diff
@@ -1,19 +1,68 @@
-<!--   Thank you for your contribution. Before you submit the pull request:
-1. Follow contributing guidelines, templates, the recommended Git workflow, and any related documentation.
-2. Read and submit the required Contributor Licence Agreements (https://github.com/kyma-project/community/blob/main/CONTRIBUTING.md#agreements-and-licenses).
+<!--   
+
+Thank you for your contribution!
+
+Before submitting your pull request, please follow these steps:
+
+1. Adhere to contributing guidelines, templates, the recommended Git workflow, and related documentation.
+2. Read and submit the required Contributor Licence Agreements (https://github.com/kyma-project/community/blob/main/docs/contributing/02-contributing.md).
 3. Test your changes and attach their results to the pull request.
 4. Update the relevant documentation.
+5. Fill in the checklist below.
```

Comment:
Below are three checklists: "Traceability", "Testability" (arguably, this one contains two lists, upping the total to four), and "Checklist". Which checklist do you mean here? Or all of them?